tests/test_tensor/test_checkpoint.py

```diff
@@ -0,0 +1,30 @@
+from torch.utils.checkpoint import checkpoint
+import torch
+from colossalai.utils import ColoInitContext
+
+
+def test_checkpoint():
+    class ModelwithCheckPoint(torch.nn.Module):
```

Comment:
move the model define to common. maybe some other places use it again.

Reply:
Why not use simpleNet and init is with checkpoint=True

Reply:
I don't think it's a right usage for checkpoint. Using CheckPointModule as the last layer is wrong because our CheckPointModule uses forward as its function, so no process_output will be invoked. If it's the last layer, grad will disappear.

Reply:
In short, I mean the class uses forward() directly instead of apply(). So it's output without grad will not be post-processed by torch itself. It can only be a middle layer of a model.

Reply:
So would you like a new class in util or an update for simple-net?

Reply:
I can use simple-net to do it. But current simple-net is wrong for checkpoint, I do some updates.